tests/test_lazy_login_and_share_times.py

```
@@ -0,0 +1,227 @@
+
+        cfg = {"base_url": "https://nas:5001", "username": "u", "password": "p"}
+        auth = MagicMock()
+        with patch("mcp_server.config.nas_configs", {"admin": cfg}), \
+             patch("mcp_server.config.get_synology_config", return_value=cfg), \
+             patch("mcp_server.SynologyAuth", return_value=auth):
+            try:
+                asyncio.run(server._handle_logout({"nas_name": "admin"}))
+            except Exception:
+                # Refusing to resolve a name with no session is fine.
+                # Authenticating in order to log out would not be.
+                pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)